ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/execution/DescribeConnectorExecutor.java

```diff
@@ -106,13 +113,39 @@ public DescribeConnectorExecutor() {
       sources = ImmutableList.of();
     }
 
+    List<KsqlWarning> warnings;
+    List<String> topics;
+    try {
+      topics = getTopics(serviceContext.getAdminClient(), connector);
+      warnings = ImmutableList.of();
+    } catch (Exception e) {
+      topics = ImmutableList.of();
+      warnings = ImmutableList.of(
+          new KsqlWarning("Could not list related topics due to " + e.getMessage())
+      );
+    }
+
     final ConnectorDescription description = new ConnectorDescription(
         configuredStatement.getStatementText(),
         info.config().get(ConnectorConfig.CONNECTOR_CLASS_CONFIG),
         status,
-        sources
+        sources,
+        topics,
+        warnings
     );
 
     return Optional.of(description);
   }
+
+  private List<String> getTopics(final Admin admin, final Optional<Connector> connector)
+      throws Exception {
+    if (!connector.isPresent()) {
+      return ImmutableList.of();
+    }
+
+    return admin.listTopics().names().get(5, TimeUnit.SECONDS)
```

Comment:
Consider making this timeout configurable?

Reply:
haha calling me out on indolence.

Reply:
So I looked into this a little bit, we don't have any configureable timeouts at the moment (in KsqlConfig anyway). I'm going to follow this PR with one that adds a `DEFAULT_EXTERNAL_SERVICE_TIMEOUT` and then we can add specific overrides for it.